android/build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven{ url "https://jitpack.io" }

```

Comment:
Google does not provide the functionality to work with USB?

Reply:
Or what is it for?

Reply:
> Google does not provide the functionality to work with USB?
I do not know. Yes, this is for USB.

Reply:
found it here:
 https://www.youtube.com/watch?v=ICjjG1VsZmw